src/core/build.ts

```
@@ -19,11 +20,16 @@ async function buildInTheBackground(service: z.infer<typeof ServiceDTO>, imageId
   if (service.containerId) {
     await removeContainer(service.containerId);
     await updateService(db, service.id, { status: "idle", containerId: null, imageId: null });
+    const providerVariables = service.providers
+      .map(provider => getProviderVariables(service.name, provider.name, provider.variables))
+      .reduce((bundle, current) => ({ ...bundle, ...current }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)